core/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -76,6 +77,7 @@ public final class OpenTelemetryRumBuilder {
     private final OtelRumConfig config;
     private final List<AndroidInstrumentation> instrumentations = new ArrayList<>();
     private final List<Consumer<OpenTelemetrySdk>> otelSdkReadyListeners = new ArrayList<>();
+    private final SessionIdTimeoutHandler timeoutHandler;
```

Comment:
We can talk more about this, because I think that we should be opinionated about sessions having a default max time to live...but the user should ultimately be able to configure and/or override this. As you pointed out, it's also consistent with the existing/current behavior.

Can we work on this aspect in a follow-up PR effort?

Reply:
Sounds good. We can follow up on it later 👍